addons/TBMod_skins/configs/CfgVehicles.hpp

```diff
@@ -126,49 +127,49 @@ class CfgVehicles
         };
     };
 
-    ///////////////////////USA Fahrzeuge//////////////////////////
+    ///////////////////////Fahrzeuge Camouflage Wüste//////////////////////////
 
-    class TB_Vehicles_USA_hunter: B_MRAP_01_F
+    class TB_Vehicles_hunter: B_MRAP_01_F
```

Comment:
Nur das du Bescheid weißt... jede Mission in der der alte Classname drin stand... verliert jetzt das Fahrzeug... was nicht schlimm ist... nur als Info... Classnames sind sehr mächtig.

Reply:
Danke für den Hinweis. Richte ich ebenso an Culli aus, auch wenn er eher selten die Modelle verwendet